src/v/config/configuration.cc

```diff
@@ -289,7 +289,13 @@ configuration::configuration()
   , topic_memory_per_partition(
       *this,
       "topic_memory_per_partition",
-      "Required memory per partition when creating topics.",
+      "Required memory in bytes per partition when creating or altering "
```

Comment:
s/partition/partition replica/ in this whole comment?

Reply:
Thanks for the note! Looks like this one:

https://github.com/redpanda-data/redpanda/pull/13409

I'll probably just submit that one separately as it doesn't actually overlap this one (textually) at all, but I've rebased it (someone added periods to all the descriptions which conflicted), and I could use your sign-off there again.

Reply:
Ooops, looks like they did overlap on exactly this message.